source/agora/common/Set.d

```diff
@@ -83,8 +83,24 @@ private void fillFrom ( T ) ( ref T[] buffer, Set!T input )
 
 /// Pick random 'count' elements from the input set,
 /// and return an array of the elements (not very memory-efficient)
```

Comment:
What about this ?

Reply:
I knew you were gonna catch this. :P

Yeah I'll add the note.